Makefile

```diff
@@ -1,11 +1,15 @@
 SKAFFOLD_VERSION=0.33.0
 CST_VERSION=1.8.0
 
-setup: .resize umask microk8s helm skaffold gitconfig ssh
+setup: .resize .profile umask microk8s helm skaffold gitconfig ssh
 
 .resize:
 	sh resize-volume.sh
 	touch .resize
+
+.profile:
+	grep -qxF 'sudo iptables -P FORWARD ACCEPT' ~/.profile || echo 'sudo iptables -P FORWARD ACCEPT' >> ~/.profile
+	grep -qxF 'umask 022' ~/.profile || echo 'umask 022' >> ~/.profile
```

Comment:
Does this replace the umask in /etc/profile.d?

Reply:
I wasn’t sure if it would but I saw a umask commented out in that file and thought it may work here instead.  I will tear it out.

Reply:
Updated to remove the other umask script after testing.